app/controllers/responses_controller.rb

```
@@ -0,0 +1,123 @@
+    @response = Response.find(params[:id])
+  end
+
+  # Finds the response map and checks that the current user owns it.
+  def find_and_authorize_map_for_create
+    @map = ResponseMap.find_by(id: params[:response_map_id])
+    unless @map
+      render json: { error: "ResponseMap not found" }, status: :not_found
+      return
+    end
+
+    unless @map.reviewer.user_id == current_user.id
```

RuboCop (1.86.1)

[convention] Use a guard clause (`return if `@map.reviewer.user_id` == current_user.id`) instead of wrapping the code inside a conditional expression.

(Style/GuardClause)